definitions/types/cidr.json

```diff
@@ -1,6 +1,10 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
   "type": "string",
+  "pattern": "^([0-9]{1,3}\\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$",
```

Comment:
this might need to support mask only cidrs (like kubernetes) `/16`

probably worth having a fullCidr / maskOnlyCidr types